rectools/model_selection/cross_validate.py

```diff
@@ -146,12 +241,14 @@ def cross_validate(  # pylint: disable=too-many-locals
                 filter_viewed=filter_viewed,
                 items_to_recommend=item_ids_to_recommend,
             )
+
             metric_values = calc_metrics(
-                metrics,
+                models_metrics[model_name],
                 reco=reco,
                 interactions=interactions_df_test,
                 prev_interactions=interactions_df_train,
                 catalog=catalog,
+                ref_reco=models_ref_reco[model_name],
```

Comment:
Here we can pass `ref_reco` from `_get_ref_reco` func output. Only filter model name that is now validated from keys of the dict.